src/assets/icons/model-3d.svg

```diff
@@ -1,5 +1,5 @@
-<svg width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" xmlns="http://www.w3.org/2000/svg">
-  <g id="icon" stroke="currentColor">
+<svg width="24" height="24" viewBox="0 0 24 24" fill="transparent" stroke="currentColor" xmlns="http://www.w3.org/2000/svg">
+  <g id="icon" stroke="currentColor" fill="transparent">
```

Comment:
I though `fill="none"` was more appropriate. What was the reason for this change?

Reply:
you're correct, all i actually needed to do was move the declaration, not make it transparent. I'll adjust.